snuba/clusters/storage_sets.py

```diff
@@ -19,7 +19,6 @@
     "SPANS": "spans",
     "EVENTS_ANALYTICS_PLATFORM": "events_analytics_platform",
     "GROUP_ATTRIBUTES": "group_attributes",
-    "METRICS_SUMMARIES": "metrics_summaries",
```

Comment:
@phacops don't you need to remove the clickhouse cluster definitions from ops (https://github.com/getsentry/ops/blob/b37ad515a21d028102dd8c9aeb8f9e751ac4b22c/k8s/services/snuba/region_overrides/us/default.yaml#L281) before you remove this from snuba?

Reply:
Good point, you're right.

Reply:
https://github.com/getsentry/ops/pull/12987

Reply:
I think when we add a new dataset, we configure the cluster and associate the dataset to the cluster even if there's no code associated with it in order to have the migrations go to the right cluster right away.

It would make sense to go the reverse order to remove the dataset, first the code, then the config in ops.